README.md

```diff
@@ -46,33 +47,52 @@ This app requires creating an app in the Azure Active Directory.
-        -   offline_access (https://graph.microsoft.com/offline_access)
-        -   User.ReadWrite.All (https://graph.microsoft.com/User.ReadWrite.All)
-        -   Calendars.ReadWrite (https://graph.microsoft.com/Calendars.ReadWrite)
-        -   OnlineMeetings.ReadWrite (https://graph.microsoft.com/OnlineMeetings.ReadWrite)
+        -   **OnlineMeetings.ReadWrite** (https://graph.microsoft.com/OnlineMeetings.ReadWrite): Allows an app to create, read online meetings on behalf of the signed-in user. Required to run the **create meeting** action.
+        -   **Calendars.ReadWrite** (https://graph.microsoft.com/Calendars.ReadWrite): Allows the app to create, read, update, and delete events in user calendars. Required to run the **create meeting** action with **add_calendar_event** set to true.
+        -   **offline_access** (https://graph.microsoft.com/offline_access): Allows the app to read and update user data, even when they are not currently using the app. This permission is required for all actions.
+        -   **Channel.ReadBasic.All** (https://graph.microsoft.com/Channel.ReadBasic.All): Read channel names and channel descriptions, on behalf of the signed-in user. This permission is required for 'list channels' action. Required to run the **list channels** action.
+        -   **User.ReadBasic.All** (https://graph.microsoft.com/User.ReadBasic.All): Allows the app to read a basic set of profile properties of other users in your organization on behalf of the signed-in user. This includes display name, first and last name, email address, open extensions and photo. Also allows the app to read the full profile of the signed-in user. Required to run the **list users, get admin consent and test connectivity action** action. To run the test connectivity and get admin consent action, User.read scope is sufficient
+        -  **ChannelMessage.Send** (https://graph.microsoft.com/ChannelMessage.Send): Allows an app to send channel messages in Microsoft Teams, on behalf of the signed-in user. Required to run the **send message** action
+        - **GroupMember.Read.All** (https://graph.microsoft.com/GroupMember.Read.All): Allows the app to list groups, read basic group properties and read membership of all groups the signed-in user has access to. Required to run the **list groups** and **list teams** action
 
         After making these changes, click **Add permissions** at the bottom of the screen, then
-        click **Grant admin consent for Phantom** .
+        click **Grant admin consent for <tenant_name>** .
+
+        For revoking the permissions, please refer [this](https://learn.microsoft.com/en-gb/azure/active-directory/manage-apps/manage-application-permissions?pivots=ms-graph) documentation.
+
+        It might happen sometimes that after removing the permissions from the azure app, it might be still visible in the state file as those permissions aren’t fully revoked.
```

Comment:
```suggestion
       After removing the permissions from the Azure app, it might still be visible in the state file as those permissions aren’t fully revoked.
```